src/utils/isTrustable.js

```diff
@@ -0,0 +1,9 @@
+export function isTrustableMapboxResult(matchCode) {
+  console.log(matchCode)
```

Comment:
Can you please move this to src/geocoder/mapbox.js as this is a function solely for the mapbox geocoder.
Would you also mind, removing the console.log() statment?

Reply:
Yes sure